docs/xxjitserveraotcachepersistence.md

```diff
@@ -0,0 +1,57 @@
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:[+|-]JITServerAOTCachePersistence
+
+**(Linux&reg; only)**
+
+This option enables or disables the JITServer server's ability to allow its later instances to reuse AOT caches.
+
+## Syntax
+
+        -XX:[+|-]JITServerAOTCachePersistence
+
+| Setting                    | Effect  | Default                                                                              |
+|----------------------------|---------|:------------------------------------------------------------------------------------:|
+|`-XX:+JITServerAOTCachePersistence` | Enable  |                                                                                      |
+|`-XX:-JITServerAOTCachePersistence` | Disable | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
+
+## Explanation
+
+ With the `-XX:+JITServerAOTCachePersistence` option, the JITServer server periodically saves its AOT caches to files, and then loads these caches from files the first time a client requests a particular named cache. It is useful to improve performance, especially in scenarios, where JITServer instances are started up and shut down regularly.
+
+ To enable this feature, you must specify the [`-XX:+JITServerUseAOTCache`](xxjitserveruseaotcache.md) command-line option, both at the server and at the client JVM.
+
+ Use the [`-XX:JITServerAOTCacheName=<cache_name>`](xxjitserveraotcachename) option to request for a particular AOT cache. If the requested cache does not exist, server checks if the reusing of AOT caches feature is enabled (`-XX:+JITServerAOTCachePersistence`), and whether a file for that cache exists. If the AOT cache file does not exist or reuse AOT cache feature is disabled (`-XX:-JITServerAOTCachePersistence`), the server creates an empty AOT cache and populates it with the compiled methods.
+
+ You can specify a directory in the [`-XX:JITServerAOTCacheDir=<directory>`](xxjitserveraotcachename.md) option to load AOT cache file from a particular directory.
```

Comment:
Maybe:
You can use the [`-XX:JITServerAOTCacheDir=<directory>`](xxjitserveraotcachename.md) option to specify the directory where the AOT cache files need to be saved to or loaded from. 
The name of an AOT cache file has the following structure: `JITServerAOTCache.<CACHE_NAME>.J<JAVA_VERSION>`, where
`CACHE_NAME` is the name of the AOT cache requested by the client and
`JAVA_VERSION` is the version of Java used by JITServer instance (example: 17 will be used for Java 17)